Doc/reference/simple_stmts.rst

```diff
@@ -667,9 +667,9 @@ The :keyword:`!break` statement
 ===============================
 
 .. index::
-   ! statement: break
-   statement: for
-   statement: while
+   ! pair: statement; break
```

Comment:
What does the `!` do?

Reply:
See https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-index

> You can mark up “main” index entries by prefixing them with an exclamation mark. The references to “main” entries are emphasized in the generated index. For example, if two pages contain

A little hard to discover, unfortunately.